src/librustc_resolve/lib.rs

```diff
@@ -4796,10 +4796,18 @@ impl<'a, 'crateloader: 'a> Resolver<'a, 'crateloader> {
                 err.span_suggestion_with_applicability(
```

Comment:
In line 4786 above, could you change the first `You` to be all lowercase?

Reply:
Also, can you apply the same fix as for `use foo as bar;` to `extern foo as bar;`?

Reply:
Not sure what you mean by your second comment ?

Reply:
The following code
```
             if let (Ok(snippet), false) = (cm.span_to_snippet(binding.span),
                                            binding.is_renamed_extern_crate()) {
```
makes it so that if it _is_ `extern foo as bar;` where `bar` already exists, we don't suggest and instead have a label with the same message. Given that you're already putting in the work to make this work for the `use foo as bar;` case, if you don't mind it'd be convenient to also fix it for the former as well.